kevm-pyk/src/kevm_pyk/utils.py

```diff
@@ -1,8 +1,56 @@
+from pathlib import Path
 from typing import Collection, Iterable, List
 
-from pyk.kast import KDefinition, KFlatModule, KImport, KTerminal, KVariable
-from pyk.kastManip import abstract_term_safely, is_anon_var, split_config_and_constraints, split_config_from, substitute
+from pyk.cfg_manager import instantiate_cell_vars, rename_generated_vars
+from pyk.cterm import CTerm
+from pyk.kast import KApply, KClaim, KDefinition, KFlatModule, KFlatModuleList, KImport, KInner, KVariable, read_kast
+from pyk.kastManip import (
+    abstract_term_safely,
+    bool_to_ml_pred,
+    bottom_up,
+    extract_lhs,
+    extract_rhs,
+    flatten_label,
+    free_vars,
+    is_anon_var,
+    ml_pred_to_bool,
+    remove_generated_cells,
+    split_config_and_constraints,
+    split_config_from,
+    substitute,
+    undo_aliases,
+)
+from pyk.kcfg import KCFG
 from pyk.ktool import KPrint
+from pyk.prelude import Bool, mlAnd
+
+
+def read_kast_flatmodulelist(ifile: Path) -> KFlatModuleList:
+    _flat_module_list = read_kast(ifile)
+    assert isinstance(_flat_module_list, KFlatModuleList)
+    return _flat_module_list
+
+
+def KCFG_from_claim(defn: KDefinition, claim: KClaim) -> KCFG:  # noqa: N802
```

Comment:
Is this following section copy-pasted from `ksummarize`? We should probably upstream this to `pyk` (with unit tests).

Reply:
Well, not quite actually. Directly from KSummarize, then minor refactor.

Reply:
It's recorded here: https://github.com/runtimeverification/ksummarize/issues/82